### PR TITLE
Adding MCloud Nemo Megatron example

### PR DESCRIPTION
**Pull request by bcui19**
We add documentation on how to run Nemo Megatron in MCloud

### CODE REVIEW

File: mcloud_examples/nemo_megatron/multi_node.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+run_name: #run-name-here
+platform: #platform-name-here
```
Comment (lupesko) [inline mcloud_examples/nemo_megatron/multi_node.yaml]:
```suggestion
cluster: #mcloud-cluster-name-here
```

File: mcloud_examples/nemo_megatron/single_node.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+run_name: #run-name-here
+platform: #platform-name-here
```
Comment (lupesko) [inline mcloud_examples/nemo_megatron/single_node.yaml]:
```suggestion
cluster: #mcloud-cluster-name-here
```

**Review by lupesko — CHANGES_REQUESTED** [review]
A few suggested updates.
I assume you ran this and ensured it works. LMK if you want to double verify, happy to run and confirm.

File: nemo/single_node.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+platform: #platform-name-here
+gpu_num: 8
+image: nvcr.io/nvidia/nemo:22.09
+env_variables:
+  - key: PYTHONUNBUFFERED
+    value: '1'
+command: | 
+
+  # getting the vocab, merge files for the tokenizer
+  wget <https://s3.amazonaws.com/models.huggingface.co/bert/gpt2-vocab.json>
```
Comment (dakinggg) [inline nemo/single_node.yaml:11]: are these commands supposed to have the `<` and `>`?

File: nemo/single_node.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  wget <https://s3.amazonaws.com/models.huggingface.co/bert/gpt2-merges.txt>
+
+  python3 /mnt/nvme/jobs/nemo/NeMo-113-release/examples/nlp/language_modeling/megatron_gpt_pretraining.py \\
```
Comment (dakinggg) [inline nemo/single_node.yaml:14]: I think these end of lines are supposed to be single slashes?

Comment (bcui19) [reply nemo/single_node.yaml:14]: Yeah, good catch sorry I was copying around things (from notion) and it added them...

File: nemo/single_node.yaml (hunk 3 above)
Comment (bcui19) [reply nemo/single_node.yaml:11]: Yeah, good catch sorry I was copying around things (from notion) and it added them...

**Review by dakinggg — APPROVED** [review]

**Review by lupesko — APPROVED** [review]

**Pull request merged**